.github/workflows/build-binaries.yaml

```
@@ -66,9 +78,9 @@ jobs:
     runs-on: ${{ inputs.runner }}
     timeout-minutes: ${{ inputs.timeout_minutes }}
     outputs:
-      publish_artifact: ${{ steps.version.outputs.publish_artifact_registry || steps.version.outputs.publish_github_release }}
+      publish_artifact_registry: ${{ steps.version.outputs.publish_artifact_registry }}
       build_version: ${{ steps.version.outputs.build_version }}
-  
+      enable_sign: ${{ steps.build.outputs.enable_sign }}
```

actionlint (1.7.11)

[error] property "build" is not defined in object type {version: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)